src/data/gateways-data-source.ts

```
@@ -0,0 +1,197 @@
+                currentHops: requestAttributesHeaders?.attributes.hops,
+              }),
+            };
+          } catch (error: any) {
+            lastError = error as Error;
+            this.log.warn('Failed to fetch from gateway', {
+              gatewayUrl,
+              priority,
+              error: error.message,
+            });
+
+            continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)